dev/tests/integration/testsuite/Magento/Review/_files/visibility_to_all_ratings.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+\Magento\TestFramework\Helper\Bootstrap::getInstance()->loadArea(
```

Comment:
Please, refactor fixtures to meet the best practices as well. It will prevent us from the additional refactoring in the future:

- Use imports
- Declare strict types
- Do not use AR model for saving (use resource model instead)

The same applies to the rollback fixture. 

Thank you!

Reply:
Also, the name of the fixture is not self-explanatory. From its name I understand that we are adding rating visible to all. But after looking at the code I see a little bit different purpose. 

Would you adjust the name, please?

Reply:
Improved both fixtures.
Thank you.